packages/blade/src/components/Button/BaseButton/__tests__/__snapshots__/BaseButton.web.test.tsx.snap

```diff
@@ -1,10 +1,53 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`<BaseButton /> should render button of type "reset" 1`] = `
+<div>
+  <button
+    class="StyledBaseButtonweb__StyledBaseButton-sc-26bt38-0 cmQBah"
+    color="hsla(213, 89%, 56%, 1)"
+    type="reset"
+  >
+    <div
```

Comment:
why is this rendering a `div` here? 🤔

Reply:
Because there's a text inside the button?